framework/packages/abstract-core/src/native/module_factory.rs

```diff
@@ -94,10 +80,6 @@ pub enum QueryMsg {
     /// Returns [`ConfigResponse`]
     #[returns(ConfigResponse)]
     Config {},
-    /// Get the installation context of the module factory.
-    /// Returns [`ContextResponse`]
-    #[returns(ContextResponse)]
-    Context {},
```

Comment:
This context is loaded by the old manager so the manager won't be able to install new modules until it updates.